testsuite/classlibrary/TestNodeProxy.sc

```diff
@@ -40,4 +40,103 @@ TestNodeProxy : UnitTest {
 		this.assertEquals(proxy.isPlaying, false, "Setting the proxy's source should not set isPlaying = true");
 	}
 
+	test_asCode_basic {
+		var codeString = "a = NodeProxy.new(Server.fromName( 'TestNodeProxy' ));\n";
+
+		this.assertEquals(proxy.asCode.postcs, codeString,
```

Comment:
remove `postcs`; also this and some other tests print `// <could not find a space for proxy: a = NodeProxy.new!>`, can you silence that somehow?

Reply:
Hm, there is a deeper problem lurking here: 
posting ```a = NodeProxy.new``` is valid code for a single proxy, 
but with more than one posted proxy, people have to  rename them meaningfully, a la
```<~proxy1> = NodeProxy.new; // <- rename me ```

When the proxy is in an anonymous proxyspace, and that proxyspace is posted, 
it would be best to only post the info line for the whole proxyspace, e.g.  like 
```
////. please create a proxyspace first, e.g. with 
//~space = ProxySpace.new(s); 
//// then rename proxies posted below: 

 ~space[<\proxy1>] = { ... }; 

opinions welcome!

Reply:
for this little PR, I would prefer to leave the post as is, 
and fix it in a later, fuller cleanup refactor.

Reply:
on second thought, changed it to clearer user info.

Reply:
asCode post of an anonymous proxy now says:
"// anonymous proxy posted - please change code to make it accessible:"
which tells users how to adapt the posted code when using it.